### PR TITLE
Css title warning fix

### PR DESCRIPTION
**Pull request by igrosso**

### CODE REVIEW

File: client/app.js [hunk 1]
```diff
@@ -23,28 +23,28 @@ const history = useRouterHistory(createHistory)({
...
-         link.type = 'text/css';
-         link.href = css;
-         link.media = 'all';
-         head.appendChild(link);
-         }
-     break;
-   default:
-     break;
-   }
+store.subscribe(function () {
```
Comment (sandrinodimattia) [inline client/app.js]: Arrow function should stay.

File: client/components/Header.jsx [hunk 2]
```diff
@@ -21,17 +21,17 @@ export default class Header extends Component {
     }
 
     return `https://cdn.auth0.com/avatars/${iss.slice(0, 2).toLowerCase()}.png`;
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Header.jsx]: Remove this.

File: client/components/Header.jsx [hunk 3]
```diff
@@ -21,17 +21,17 @@ export default class Header extends Component {
...
-    const { user, issuer, onLogout, accessLevel, settings } = this.props;
+    const { user, issuer, onLogout, accessLevel } = this.props;
     const showMenu = accessLevel.role === 2;
     return (
       <header className="dashboard-header">
         <nav role="navigation" className="navbar navbar-default">
           <div className="container">
             <div id="header" className="navbar-header" style={{ width: '800px' }}>
-              <a className="navbar-brand" href="#">{settings.get('settings').get('title') || window.config.TITLE}</a>
+              <a className="navbar-brand" href="#">{window.config.TITLE}</a>
```
Comment (sandrinodimattia) [inline client/components/Header.jsx]: You removed my change here. The title from settings has priority.

File: client/components/Users/UserActions.jsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ export default class UserActions extends Component {
     resetPassword: PropTypes.func.isRequired,
     unblockUser: PropTypes.func.isRequired,
     user: PropTypes.object.isRequired
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 5]
```diff
@@ -117,7 +117,7 @@ export default class UserActions extends Component {
         Remove MFA ({user.multifactor[0]})
       </MenuItem>
     );
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 6]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
         Block User
       </MenuItem>
     );
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 7]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   deleteUser = () => {
     this.props.deleteUser(this.state.user);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 8]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   resetPassword = () => {
     this.props.resetPassword(this.state.user, this.state.databaseConnections[0]);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 9]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   changePassword = () => {
     this.props.changePassword(this.state.user, this.state.databaseConnections[0]);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 10]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   changeUsername = () => {
     this.props.changeUsername(this.state.user, this.state.databaseConnections[0]);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 11]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   changeEmail = () => {
     this.props.changeEmail(this.state.user, this.state.databaseConnections[0]);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 12]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   resendVerificationEmail = () => {
     this.props.resendVerificationEmail(this.state.user, this.state.databaseConnections[0]);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 13]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   blockUser = () => {
     this.props.blockUser(this.state.user);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 14]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   unblockUser = () => {
     this.props.unblockUser(this.state.user);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserActions.jsx [hunk 15]
```diff
@@ -133,43 +133,43 @@ export default class UserActions extends Component {
...
 
   removeMfa = () => {
     this.props.removeMfa(this.state.user, this.state.user.multifactor[0]);
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserActions.jsx]: Unnecessary semicolon

File: client/components/Users/UserDevices.jsx [hunk 16]
```diff
@@ -1,13 +1,23 @@
...
+} from '../Dashboard';
 
 export default class UserDevices extends Component {
   static propTypes = {
     user: PropTypes.object,
     error: PropTypes.string,
     devices: PropTypes.object.isRequired,
     loading: PropTypes.bool.isRequired
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserDevices.jsx]: Unnecessary semicolon

File: client/components/Users/UserForm.jsx [hunk 17]
```diff
@@ -15,22 +15,21 @@ export default createForm('user', class extends Component {
     userWasSaved: React.PropTypes.func.isRequired,
     fetchUsers: React.PropTypes.func.isRequired,
     title: React.PropTypes.string.isRequired,
-    closeConfirmation: React.PropTypes.func.isRequired,
     confirmLoading: PropTypes.bool.isRequired,
     hideConfirmWindow: PropTypes.func.isRequired,
     userCreateError: PropTypes.string
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Unnecessary semicolon

File: client/components/Users/UserForm.jsx [hunk 18]
```diff
@@ -42,7 +41,7 @@ export default createForm('user', class extends Component {
       if (this.refs.password.props.field.value !== this.refs.repeat_password.props.field.value) {
         this.setState({
           customErrors: {
-            repeat_password: [ 'Repeat Password must be equal to password' ]
+            repeat_password: ['Repeat Password must be equal to password']
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Spaces are required here.

File: client/components/Users/UserForm.jsx [hunk 19]
```diff
@@ -82,7 +84,7 @@ export default createForm('user', class extends Component {
         usernameRequired: false
       });
     }
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Unnecessary semicolon

File: client/components/Users/UserForm.jsx [hunk 20]
```diff
@@ -123,28 +118,36 @@ export default createForm('user', class extends Component {
...
+      confirmLoading, userCreateError
+    } = this.props;
     const options = this.getOptions(memberships);
     return (
-      <Confirm title={title} show={show} loading={confirmLoading} onCancel={this.props.hideConfirmWindow} onConfirm={this.onConfirmUserCreate}>
-        <Error message={userCreateError} />
+      <Confirm title={title} show={show} loading={confirmLoading} onCancel={this.props.hideConfirmWindow}
+               onConfirm={function () {
+                 this.onConfirmUserCreate(options);
+               }.bind(this)}>
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Don't use bind.

File: client/components/Users/UserForm.jsx [hunk 21]
```diff
@@ -123,28 +118,36 @@ export default createForm('user', class extends Component {
...
+        <Error message={userCreateError}/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Space before closing tag

File: client/components/Users/UserForm.jsx [hunk 22]
```diff
@@ -123,28 +118,36 @@ export default createForm('user', class extends Component {
...
         <div className="row">
           <form className="createUserScreenForm form-horizontal col-xs-12" style={{ marginTop: '30px' }}>
             <div className="custom_field">
-              <InputText field={email} fieldName="email" label="Email" validationErrors={validationErrors} ref="email" />
+              <InputText field={email} fieldName="email" label="Email" validationErrors={validationErrors} ref="email"/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Space before closing tag

File: client/components/Users/UserForm.jsx [hunk 23]
```diff
@@ -123,28 +118,36 @@ export default createForm('user', class extends Component {
...
             </div>
             {usernameRequired ?
               <div className="custom_field">
-                <InputText field={username} fieldName="username" label="username" validationErrors={validationErrors} ref="username" />
+                <InputText field={username} fieldName="username" label="username" validationErrors={validationErrors}
+                           ref="username"/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Space before closing tag

File: client/components/Users/UserForm.jsx [hunk 24]
```diff
@@ -123,28 +118,36 @@ export default createForm('user', class extends Component {
...
               </div>
               : ''}
             <div className="custom_field">
-              <InputText field={password} fieldName="password" label="Password" type="password" validationErrors={validationErrors} ref="password" />
+              <InputText field={password} fieldName="password" label="Password" type="password"
+                         validationErrors={validationErrors} ref="password"/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Space before closing tag

File: client/components/Users/UserForm.jsx [hunk 25]
```diff
@@ -123,28 +118,36 @@ export default createForm('user', class extends Component {
...
             </div>
             <div className="custom_field repeat_password">
-              <InputText field={repeat_password} fieldName="repeat_password" label="Repeat Password" type="password" validationErrors={this.state.customErrors} ref="repeat_password" />
+              <InputText field={repeat_password} fieldName="repeat_password" label="Repeat Password" type="password"
+                         validationErrors={this.state.customErrors} ref="repeat_password"/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Space before closing tag

File: client/components/Users/UserHeader.jsx [hunk 26]
```diff
@@ -5,7 +5,7 @@ export default class UserHeader extends Component {
     error: PropTypes.string,
     loading: PropTypes.bool.isRequired,
     user: PropTypes.object.isRequired
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserHeader.jsx]: Unnecessary semicolon

File: client/components/Users/UserInfo.jsx [hunk 27]
```diff
@@ -1,17 +1,14 @@
...
 
 export default class UserInfo extends Component {
   static propTypes = {
     error: PropTypes.string,
     loading: PropTypes.bool.isRequired,
     user: PropTypes.object.isRequired,
-    memberships: PropTypes.array
-  }
+    memberships: PropTypes.object
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserInfo.jsx]: Unnecessary semicolon

File: client/components/Users/UserInfo.jsx [hunk 28]
```diff
@@ -20,38 +17,49 @@ export default class UserInfo extends Component {
   getDepartments = (memberships) => {
     const meta = memberships || [];
     return meta.join(', ');
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserInfo.jsx]: Unnecessary semicolon

File: client/components/Users/UserInfo.jsx [hunk 29]
```diff
@@ -20,38 +17,49 @@ export default class UserInfo extends Component {
...
 
   getIdentities = (user) => {
     if (user.size === 0) return {};
     return user.get('identities').toJSON()[0];
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserInfo.jsx]: Unnecessary semicolon

File: client/components/Users/UserInfo.jsx [hunk 30]
```diff
@@ -20,38 +17,49 @@ export default class UserInfo extends Component {
...
 
   getBlocked = (user) => {
     if (user.size === 0) return '';
     return user.get('blocked') ? 'Yes' : 'No';
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserInfo.jsx]: Unnecessary semicolon

File: client/components/Users/UserInfo.jsx [hunk 31]
```diff
@@ -20,38 +17,49 @@ export default class UserInfo extends Component {
...
-            <UserInfoField title="Email">{user.get('email')}</UserInfoField>
-            <UserInfoField title="Identity">{identity.connection}</UserInfoField>
-            <UserInfoField title="Blocked">{blocked}</UserInfoField>
-            <UserInfoField title="Last IP">{user.get('last_ip')}</UserInfoField>
-            <UserInfoField title="Logins Count">{user.get('logins_count')}</UserInfoField>
-            <UserInfoField title="Memberships">{departments}</UserInfoField>
-            <UserInfoField title="Signed Up">{moment(user.get('created_at')).fromNow()}</UserInfoField>
-            <UserInfoField title="Updated">{moment(user.get('updated_at')).fromNow()}</UserInfoField>
-            <UserInfoField title="Last Login">{moment(user.get('last_login')).fromNow()}</UserInfoField>
+          <div>
```
Comment (sandrinodimattia) [inline client/components/Users/UserInfo.jsx]: You removed all of my changes. Don't.

File: client/components/Users/UserLogs.jsx [hunk 32]
```diff
@@ -8,7 +8,7 @@ export default class UserLogs extends Component {
     loading: React.PropTypes.bool.isRequired,
     user: React.PropTypes.object.isRequired,
     logs: React.PropTypes.object.isRequired
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserLogs.jsx]: Unnecessary semicolon

File: client/components/Users/UserOverview.jsx [hunk 33]
```diff
@@ -11,36 +11,34 @@ export default class UserOverview extends React.Component {
     error: React.PropTypes.object,
     users: React.PropTypes.array.isRequired,
     total: React.PropTypes.number.isRequired,
-    loading: React.PropTypes.bool.isRequired,
-    renderActions: React.PropTypes.func.isRequired
-  }
+    loading: React.PropTypes.bool.isRequired
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserOverview.jsx]: Unnecessary semicolon

File: client/components/Users/UserOverview.jsx [hunk 34]
```diff
@@ -11,36 +11,34 @@ export default class UserOverview extends React.Component {
...
 
   onKeyPress = (e) => {
     if (e.key === 'Enter') {
       this.props.onSearch(findDOMNode(this.refs.search).value);
     }
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserOverview.jsx]: Unnecessary semicolon

File: client/components/Users/UserOverview.jsx [hunk 35]
```diff
@@ -11,36 +11,34 @@ export default class UserOverview extends React.Component {
...
-              <Error message={error} />
+              <Error message={error}/>
             </div>
           </div>
-          <SearchBar onReset={this.props.onReset} onSearch={this.props.onSearch} enabled={!loading} />
+          <SearchBar onReset={this.props.onReset} onSearch={this.props.onSearch} enabled={!loading}/>
           <div className="row">
             <div className="col-xs-12">
-              <UsersTable loading={loading} users={users} renderActions={renderActions} />
+              <UsersTable loading={loading} users={users}/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserOverview.jsx]: Space before closing tag

File: client/components/Users/UserOverview.jsx [hunk 36]
```diff
@@ -11,36 +11,34 @@ export default class UserOverview extends React.Component {
...
             </div>
           </div>
           <div className="row">
             <div className="col-xs-12">
-              <TableTotals currentCount={users.length} totalCount={total} />
+              <TableTotals currentCount={users.length} totalCount={total}/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserOverview.jsx]: Space before closing tag

File: client/components/Users/UserProfile.jsx [hunk 37]
```diff
@@ -6,7 +6,7 @@ export default class UserProfile extends Component {
     error: PropTypes.string,
     loading: PropTypes.bool.isRequired,
     user: PropTypes.object.isRequired
-  }
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UserProfile.jsx]: Unnecessary semicolon

File: client/components/Users/UserProfile.jsx [hunk 38]
```diff
@@ -17,7 +17,7 @@ export default class UserProfile extends Component {
     return (
       <LoadingPanel show={loading} animationStyle={{ paddingTop: '5px', paddingBottom: '5px' }}>
         <Error message={error}>
-          <Json jsonObject={user.toJS()} />
+          <Json jsonObject={user.toJS()}/>
```
Comment (sandrinodimattia) [inline client/components/Users/UserProfile.jsx]: Space before closing tag

File: client/components/Users/UsersTable.jsx [hunk 39]
```diff
@@ -1,43 +1,53 @@
...
+} from '../Dashboard';
 
 export default class UsersTable extends Component {
   static propTypes = {
     users: React.PropTypes.array.isRequired,
-    loading: React.PropTypes.bool.isRequired,
-    renderActions: React.PropTypes.func.isRequired
-  }
+    loading: React.PropTypes.bool.isRequired
+  };
```
Comment (sandrinodimattia) [inline client/components/Users/UsersTable.jsx]: Unnecessary semicolon

File: client/components/Users/UsersTable.jsx [hunk 40]
```diff
@@ -1,43 +1,53 @@
...
   }
 
   render() {
-    const { users, renderActions } = this.props;
+    const { users } = this.props;
     return (
       <Table>
         <TableHeader>
-          <TableColumn width="6%" />
+          <TableColumn width="6%"/>
```
Comment (sandrinodimattia) [inline client/components/Users/UsersTable.jsx]: Space before closing tag

File: client/components/Users/UsersTable.jsx [hunk 41]
```diff
@@ -1,43 +1,53 @@
...
-            <TableTextCell>{user.last_login_relative}</TableTextCell>
-            <TableTextCell>{user.logins_count}</TableTextCell>
-            <TableTextCell>{user.identities[0].connection}</TableTextCell>
-          </TableRow>
-        )}
+          {users.map((user, index) =>
+            <TableRow key={index}>
+              <TableCell>
+                <img className="img-circle" src={user.picture} alt={user.name || user.email || user.user_id}
+                     width="32"/>
```
Comment (sandrinodimattia) [inline client/components/Users/UsersTable.jsx]: Space before closing tag

File: client/containers/Logs.jsx [hunk 42]
```diff
@@ -21,7 +21,7 @@ class LogsContainer extends Component {
     this.props.fetchLogs(this.props.logs.nextPage);
   }
 
-  createToolbar(isBottom: false) {
+  createToolbar(isBottom:false) {
```
Comment (sandrinodimattia) [inline client/containers/Logs.jsx]: Space after colon

File: client/containers/Logs.jsx [hunk 43]
```diff
@@ -41,23 +41,24 @@ class LogsContainer extends Component {
     return (
       <div>
         <TabsHeader role={ accessLevel.role }/>
-        <LogDialog onClose={this.props.clearLog} error={log.error} loading={log.loading} log={log.record} logId={log.id} />
+        <LogDialog onClose={this.props.clearLog} error={log.error} loading={log.loading} log={log.record}
+                   logId={log.id}/>
```
Comment (sandrinodimattia) [inline client/containers/Logs.jsx]: Space after closing tag

File: client/containers/Logs.jsx [hunk 44]
```diff
@@ -41,23 +41,24 @@ class LogsContainer extends Component {
...
         <div className="row">
           <div className="col-xs-12 wrapper">
             {this.createToolbar(false)}
           </div>
         </div>
         <div className="row">
           <div className="col-xs-12 wrapper">
-            <Error message={logs.error} />
+            <Error message={logs.error}/>
```
Comment (sandrinodimattia) [inline client/containers/Logs.jsx]: Space after closing tag

File: client/containers/Logs.jsx [hunk 45]
```diff
@@ -41,23 +41,24 @@ class LogsContainer extends Component {
...
             <LoadingPanel show={logs.loading}>
-              <LogsTable onOpen={this.props.fetchLog} loading={logs.loading} logs={logs.records} />
+              <LogsTable onOpen={this.props.fetchLog} loading={logs.loading} logs={logs.records}/>
```
Comment (sandrinodimattia) [inline client/containers/Logs.jsx]: Space after closing tag

File: client/containers/Logs.jsx [hunk 46]
```diff
@@ -41,23 +41,24 @@ class LogsContainer extends Component {
...
             </LoadingPanel>
           </div>
         </div>
         <div className="row">
           <div className="col-xs-12 wrapper">
-            <TableTotals currentCount={logs.records.size} totalCount={logs.total} />
+            <TableTotals currentCount={logs.records.size} totalCount={logs.total}/>
```
Comment (sandrinodimattia) [inline client/containers/Logs.jsx]: Space before closing tag

File: client/containers/Users/User.css [hunk 47]
```diff
@@ -94,3 +94,26 @@
   position: relative;
   bottom: 5px;
 }
+.userInfoBlock{
```
Comment (sandrinodimattia) [inline client/containers/Users/User.css]: Remove these, I deleted them, don't re-add them.

File: client/containers/Users/User.jsx [hunk 48]
```diff
@@ -40,51 +35,55 @@ export default connectContainer(class extends Component {
     const { user, databaseConnections, log, logs, devices } = this.props;
     return (
       <div className="user">
-        <TabsHeader role={this.props.accessLevel.role} />
+        <TabsHeader role={this.props.accessLevel.role}/>
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: Space

File: client/containers/Users/User.jsx [hunk 49]
```diff
@@ -40,51 +35,55 @@ export default connectContainer(class extends Component {
...
           </div>
         </div>
         <div className="row user-tabs">
           <div className="col-xs-12">
-            <Tabs defaultActiveKey={1} animation={false}>
+            <Tabs defaultActiveKey={1} animation={false} id="userViewTabs">
               <Tab eventKey={1} title="User Information">
-                <UserInfo loading={user.get('loading')} user={user.get('record')} memberships={user.get('memberships')} error={user.get('error')} />
+                <UserInfo loading={user.get('loading')} user={user.get('record')} memberships={user.get('memberships')}
+                          error={user.get('error')}/>
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: Space before closing tag

File: client/containers/Users/User.jsx [hunk 50]
```diff
@@ -40,51 +35,55 @@ export default connectContainer(class extends Component {
...
+                  resendVerificationEmail={this.props.requestResendVerificationEmail}
+                />
+                : ''}
             </div>
           </div>
         </div>
         <div className="row">
           <div className="col-xs-12">
-            <UserHeader loading={user.get('loading')} user={user.get('record')} error={user.get('error')} />
+            <UserHeader loading={user.get('loading')} user={user.get('record')} error={user.get('error')}/>
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: Space before closing tag

File: client/containers/Users/User.jsx [hunk 51]
```diff
@@ -40,51 +35,55 @@ export default connectContainer(class extends Component {
...
+            <Tabs defaultActiveKey={1} animation={false} id="userViewTabs">
               <Tab eventKey={1} title="User Information">
-                <UserInfo loading={user.get('loading')} user={user.get('record')} memberships={user.get('memberships')} error={user.get('error')} />
+                <UserInfo loading={user.get('loading')} user={user.get('record')} memberships={user.get('memberships')}
+                          error={user.get('error')}/>
               </Tab>
               <Tab eventKey={2} title="Devices">
-                <UserDevices loading={devices.get('loading')} devices={devices.get('records')} error={devices.get('error')} />
+                <UserDevices loading={devices.get('loading')} devices={devices.get('records')}
+                             error={devices.get('error')}/>
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: Space before closing tag

File: client/containers/Users/User.jsx [hunk 52]
```diff
@@ -40,51 +35,55 @@ export default connectContainer(class extends Component {
...
               </Tab>
               <Tab eventKey={3} title="Logs">
                 <LogDialog onClose={this.props.clearLog} error={log.get('error')}
                            loading={log.get('loading')} log={log.get('record')}
-                           logId={log.get('logId')} />
+                           logId={log.get('logId')}/>
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: Space before closing tag

File: client/containers/Users/User.jsx [hunk 53]
```diff
@@ -40,51 +35,55 @@ export default connectContainer(class extends Component {
...
                 <UserLogs onOpen={this.props.fetchLog} loading={logs.get('loading')}
                           logs={logs.get('records')} user={user.get('record')}
-                          error={logs.get('error')} />
+                          error={logs.get('error')}/>
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: Space before closing tag

File: client/containers/Users/User.jsx [hunk 54]
```diff
@@ -40,51 +35,55 @@ export default connectContainer(class extends Component {
...
               </Tab>
               <Tab eventKey={4} title="Profile">
-                <UserProfile loading={user.get('loading')} user={user.get('record')} error={user.get('error')} />
+                <UserProfile loading={user.get('loading')} user={user.get('record')} error={user.get('error')}/>
```
Comment (sandrinodimattia) [inline client/containers/Users/User.jsx]: Space before closing tag

File: client/components/Users/UserForm.jsx [hunk 55]
```diff
@@ -15,22 +15,21 @@ export default createForm('user', class extends Component {
...
-  }
+  };
 
   constructor(props) {
     super(props);
     this.state = {
       usernameRequired: false,
       memberships: false,
-      customErrors: { }
+      customErrors: {}
```
Comment (sandrinodimattia) [inline client/components/Users/UserForm.jsx]: Space is required here

File: client/containers/Configuration/Configuration.jsx [hunk 56]
```diff
@@ -69,11 +69,11 @@ class Configuration extends Component {
           </div>
           <div className="row user-tabs">
             <div className="col-xs-12">
-              <Tabs defaultActiveKey={this.state.activeTab} animation={false}>
+              <Tabs defaultActiveKey={this.state.activeTab} animation={false} id="configurationTabs">
                 <Tab eventKey={1} title="Filter Hook">
                   <p>The <strong>filter hook</strong> allows you to specify which records are shown to the current users when loading the list of users or searching. For example: <i>Only show users from my department</i>. This has to be defined using the lucene syntax.</p>
                   <Editor
-                    value={this.state.scripts.filter}
+                    value={String(this.state.scripts.filter)}
```
Comment (sandrinodimattia) [inline client/containers/Configuration/Configuration.jsx]: Why?

File: client/containers/Configuration/Configuration.jsx [hunk 57]
```diff
@@ -69,11 +69,11 @@ class Configuration extends Component {
           </div>
           <div className="row user-tabs">
             <div className="col-xs-12">
-              <Tabs defaultActiveKey={this.state.activeTab} animation={false}>
+              <Tabs defaultActiveKey={this.state.activeTab} animation={false} id="configurationTabs">
```
Comment (sandrinodimattia) [inline client/containers/Configuration/Configuration.jsx]: configuration-tabs

File: client/containers/Configuration/Configuration.jsx (hunk 56 above)
Comment (igrosso) [inline client/containers/Configuration/Configuration.jsx]: Editor props.type validation
  value: PropTypes.string.isRequired,
Without String() type is function

**Pull request closed without merging**